odoo/tools/config.py

```diff
@@ -322,6 +325,10 @@ def __init__(self, fname=None):
                              help="Maximum allowed Real time per cron job. (default: --limit-time-real). "
                                   "Set to 0 for no limit. ",
                              type="int")
+            group.add_option("--limit-time-real-async", dest="limit_time_real_async", my_default=-1,
+                             help="Maximum allowed Real time per async job. (default: --limit-time-real). "
+                                  "Set to 0 for no limit. ",
+                             type="int")
```

Comment:
Wouldn't it make sense for the async limit to be the same as the cron if unspecified?

Reply:
what about `raise NotImplementedError` ?